app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -552,23 +552,20 @@ function useGrantDetail() {
 
     if (owner !== g.owner && payee === g.payee && !isMetaPtrUpdated) {
       // update Grant Owner
-      tx = await registry.updateGrantOwner(g.id, owner);
+      txCall = () => registry.updateGrantOwner(g.id, owner);
     } else if (owner === g.owner && payee !== g.payee && !isMetaPtrUpdated) {
       // update Grant Payee
-      tx = await registry.updateGrantPayee(g.id, payee);
+      txCall = () => registry.updateGrantPayee(g.id, payee);
     } else if (owner === g.owner && payee === g.payee && isMetaPtrUpdated) {
       // update Grant MetaPtr
-      tx = await registry.updateGrantMetaPtr(g.id, metaPtr);
+      txCall = () => registry.updateGrantMetaPtr(g.id, metaPtr);
     } else {
       // update all
-      tx = await registry.updateGrant(g.id, owner, payee, metaPtr);
+      txCall = () => registry.updateGrant(g.id, owner, payee, metaPtr);
```

Comment:
Could you document somewhere—even if just with comments where we define `watchTransaction`—what example usage should look like with some pseudocode/code snippets in the comments? Just since this is a different flow than we had before and I want to make sure it's easy to understand without having to find actual usage of the method